### PR TITLE
Continue loading data files if a single file cannot be loaded.

### PR DESCRIPTION
**Pull request by krzywon**
This is a cherry pick from ticket-1111.

### CODE REVIEW

**Comment by butlerpd** [comment]
ready for testing on Win64

**Comment by butlerpd** [comment]
ready for testing on Win64

File: src/sas/sasgui/guiframe/local_perspectives/data_loader/data_loader.py [hunk 1]
```diff
@@ -204,7 +204,7 @@ def get_data(self, path, format=None):
                                  info="info")
             except NoKnownLoaderException as e:
                 exception_occurred = True
-                error_message = "Loading data failed!\n" + e.message
+                error_message = "Loading data failed!" + e.message
```
Comment (pkienzle) [inline src/sas/sasgui/guiframe/local_perspectives/data_loader/data_loader.py]: Don't you need a space after `failed!`? Or do you know that e.message starts with a space?

Comment (krzywon) [reply src/sas/sasgui/guiframe/local_perspectives/data_loader/data_loader.py]: A NoKnownLoaderException message always starts with a leading carriage return.

**Pull request merged**